Dockerfile

```diff
@@ -1,33 +1,34 @@
-FROM elixir:1.9.0-alpine AS build
+FROM hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3 AS build
 
-RUN apk add --no-cache build-base npm git python
+RUN apk add --update git nodejs=14.16.1-r1 npm
 WORKDIR /app
 
 RUN mix local.hex --force && \
     mix local.rebar --force
 
 ENV MIX_ENV=prod
+ENV NODE_ENV=prod
 
 COPY mix.exs mix.lock ./
 COPY config config
 COPY .version ./
-RUN mix deps.get 
-RUN mix deps.compile
+RUN mix deps.get --force --only prod
 
 COPY assets/package.json assets/package-lock.json ./assets/
-RUN npm --prefix ./assets ci --progress=false --no-audit --loglevel=error
+RUN cd assets && npm install
 
 COPY priv priv
 COPY assets assets
 RUN npm run --prefix ./assets deploy
-RUN mix phx.digest
+RUN mix phx.digest && \
+    mix release
```

Comment:
no need of release here. release should include the `lib` folder which is next step